corellium/adapter/src/test/kotlin/flank/corellium/adapter/Example.kt

```diff
@@ -85,8 +86,8 @@ fun main() {
         )
 
         println("* Executing tests")
-        api.executeTest(testPlan).collect { line ->
-            print(line)
+        api.executeTest(testPlan).forEach { flow ->
+            launch { flow.collect { line -> println(line) } }
         }
```

Comment:
is there any reason why you did not merge flow? multiple collects is not so common way to do such operation